tools/prepare-source.sh

```diff
@@ -13,8 +13,11 @@ if test "$LATEST" != "$GO_VERSION"; then
 fi
 
 set -x
+export GOTOOLCHAIN=go$GO_VERSION
+export GOSUMDB='sum.golang.org' # this is turned off for Go from Fedora / RHEL
+go version
```

Comment:
This ensures the Go version in this script is executed with the exactly same version as intendent. The SUM thing is a Fedora custom Go configuration which brakes this locally - sumdb is turned off for Fedora.